bpf/tests/bpf_nat_tests.c

```diff
@@ -605,7 +605,7 @@ int test_nat4_icmp_error_tcp_egress(__maybe_unused struct __ctx_buff *ctx)
 	/* This is the entry-point of the test, calling
 	 * snat_v4_nat().
 	 */
-	ret = snat_v4_nat(ctx, &icmp_tuple, ip4, l4_off, &target, &trace, NULL);
+	ret = snat_v4_nat(ctx, &icmp_tuple, ip4, 0, l4_off, &target, &trace, NULL);
```

Comment:
(non-blocking) as we have an `ip4`, could we also pass an actual `fraginfo` here? Or is the header malformed in some way?